test/unit/_helpers.bash

```diff
@@ -2,3 +2,41 @@
 chart_dir() {
     echo ${BATS_TEST_DIRNAME}/../..
 }
+
+# helm is used to intercept the helm command in tests and change flags depending
```

Comment:
```suggestion
# helm is used to intercept the Helm command in tests and change flags depending
```

Reply:
I'm gonna use
```
`helm`
```

here to be clear that we're intercepting the literal helm string.